app/Services/VulnTogglesService.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Services;
+
+use App\Http\Requests\VulnTogglesRequest;
+
+class VulnTogglesService
+{
+    public function updateVulnToggles(VulnTogglesRequest $request)
+    {
+        $toggle = $request->input('toggle');
+        $value = $request->input('value') ? true : false;
+        if (array_key_exists($toggle, auth()->user()->getAttributes())) {
+            auth()->user()->{$toggle} = $value;
+            auth()->user()->save();
+            return response()->json(['success' => true]);
+        } else {
+            return response()->json(['success' => false]);
```

Comment:
Consider including a useful error message on failure. E.g.,
```suggestion
            return response()->json(['success' => false, 'error' => 'Invalid toggle name'], 422);
```